styles.css

```diff
@@ -812,10 +924,120 @@ ul {
     color: #5e6c84;
   }
 
-  /* .blue-btn {
-    background-color: #4053fc;
-    color: #fff;
-  } */
+  #popup {
+    display: none;
+  }
+
+  .popup {
+    display: flex;
+    border-radius: 16px;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 80%;
+    height: auto;
+    z-index: 100;
+    background-color: white;
+    position: fixed;
+    top: 50px;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    border: thin solid black;
+    margin: 0 auto;
+  }
```

Comment:
- [ ] Please kindly increase the margin-bottom of your pop-up window for desktop version and kindly ensure that all the elements are properly displayed and aligned according to the template. 👍 
- [ ] Moreso, the text and image in the button elements too are not properly placed. 👍 

## Your Design
<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225601107-87b9e8e6-db21-4713-b22f-02dc5a2303bf.png">

## Figma Design
<img width="435" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225600708-156672b1-bfa9-43a6-a16b-c8a7cf597ade.png">